src/main/java/com/hida/Minter.java

```diff
@@ -0,0 +1,576 @@
+        this.Prefix = Prefix;
+
+        this.RootLength = CharMap.length();
+
+        if (sansVowel) {
+            this.BASE_MAP.put('d', DIGIT_TOKEN);
+            this.BASE_MAP.put('l', SANS_VOWEL_TOKEN);
+            this.BASE_MAP.put('u', SANS_VOWEL_TOKEN.toUpperCase());
+            this.BASE_MAP.put('m', SANS_VOWEL_TOKEN + SANS_VOWEL_TOKEN.toUpperCase());
+            this.BASE_MAP.put('e', DIGIT_TOKEN + SANS_VOWEL_TOKEN + SANS_VOWEL_TOKEN.toUpperCase());
+        } else {
+            this.BASE_MAP.put('d', DIGIT_TOKEN);
+            this.BASE_MAP.put('l', VOWEL_TOKEN);
+            this.BASE_MAP.put('u', VOWEL_TOKEN.toUpperCase());
+            this.BASE_MAP.put('m', VOWEL_TOKEN + VOWEL_TOKEN.toUpperCase());
+            this.BASE_MAP.put('e', DIGIT_TOKEN + VOWEL_TOKEN + VOWEL_TOKEN.toUpperCase());
+        }
+    }
+
+    /**
+     * 
+     * @param token
+     * @return 
+     */
+    public String getMap(String token) {
+        String map;
+        if (token.equals("DIGIT")) {
+            map = DIGIT_TOKEN;
+        } else if (token.equals("LOWERCASE")) {
+            map = VOWEL_TOKEN;
+        } else if (token.equals("UPPERCASE")) {
+            map = VOWEL_TOKEN.toUpperCase();
+        } else if (token.equals("MIXEDCASE")) {
+            map = VOWEL_TOKEN + VOWEL_TOKEN.toUpperCase();
+        } else if (token.equals("LOWER_EXTENDED")) {
+            map = DIGIT_TOKEN + VOWEL_TOKEN;
+        } else if (token.equals("UPPER_EXTENDED")) {
+            map = DIGIT_TOKEN + VOWEL_TOKEN.toUpperCase();
+        } else if (token.equals("MIXED_EXTENDED")) {
+            map = DIGIT_TOKEN + VOWEL_TOKEN + VOWEL_TOKEN.toUpperCase();
+        } else if (token.equals("SANS_VOWEL_LOWER")) {
+            map = SANS_VOWEL_TOKEN;
+        } else if (token.equals("SANS_VOWEL_UPPER")) {
+            map = SANS_VOWEL_TOKEN.toUpperCase();
+        } else if (token.equals("SANS_VOWEL_MIXED")) {
+            map = SANS_VOWEL_TOKEN + SANS_VOWEL_TOKEN.toUpperCase();
+        } else if (token.equals("SANS_VOWEL_LOWER_EXTENDED")) {
+            map = DIGIT_TOKEN + SANS_VOWEL_TOKEN;
+        } else if (token.equals("SANS_VOWEL_UPPER_EXTENDED")) {
+            map = DIGIT_TOKEN + SANS_VOWEL_TOKEN.toUpperCase();
+        } else if (token.equals("SANS_VOWEL_MIXED_EXTENDED")) {
+            map = DIGIT_TOKEN + SANS_VOWEL_TOKEN + SANS_VOWEL_TOKEN.toUpperCase();
+        } else {
+            map = token;
+        }
+        return map;
+    }
+
+    /**
+     * Continuously increments ids until a unique id has been found.
+     * @param original
+     * @param order
+     * @param isAuto
+     * @param amount
+     * @return
+     * @throws SQLException 
+     */
+    private Set<Id> rollIds(Set<Id> original, boolean order, boolean isAuto, int amount)
+            throws SQLException {
+        Set<Id> duplicateCache = new HashSet(amount);
+        duplicateCache.addAll(original);
+
+        // finds a list of ids that is unique to the database            

```

Comment:
This comment doesn't seem to be in line with what the method actually does (e.g. it returns a `boolean` not a collection)